simulation/game-of-life/src/app/components/simulation/simulation.component.html

```diff
@@ -23,4 +23,5 @@
     <button (click)="stop()">Stop</button>
     <button (click)="reset()">Reset</button>
   </div>
-</app-content>
\ No newline at end of file
+  <h3>Use the shift key to make lines</h3>
```

Comment:
Will try to put these lines in an instructions menu, but not sure yet how this would look like